src/front/wgsl/mod.rs

```diff
@@ -326,11 +327,16 @@ impl<'a> Error<'a> {
                 labels: vec![(span.clone(), "unknown local function".into())],
                 notes: vec![],
             },
-            Error::LetTypeMismatch(ref name_span, ref expected_ty) => ParseError {
+            Error::InitializationTypeMismatch(ref name_span, ref expected_ty) => ParseError {
                 message: format!("the type of `{}` is expected to be {:?}", &source[name_span.clone()], expected_ty),
                 labels: vec![(name_span.clone(), format!("definition of `{}`", &source[name_span.clone()]).into())],
                 notes: vec![],
             },
+            Error::MissingType(ref name_span) => ParseError {
```

Comment:
nit: would be useful to have a snapshot test for this too

Reply:
good point, adding now!